api/v1beta1/validations.go

```diff
@@ -118,6 +123,120 @@ func ValidateCount(count *intstr.IntOrString) error {
 	return nil
 }
 
+func newGoValidator() (*validator.Validate, ut.Translator, error) {
+	englishLocale := en.New()
+	uni := ut.New(englishLocale, englishLocale)
+
+	translator, _ := uni.GetTranslator("en")
+
+	validate := validator.New()
```

Comment:
Small note related to the following:
> Special Notes:
  If new to using validator it is highly recommended to initialise it using the WithRequiredStructEnabled option which is opt-in to new behaviour that will become the default behaviour in v11+. See documentation for more details.
  `validate := validator.New(validator.WithRequiredStructEnabled())`

https://pkg.go.dev/github.com/go-playground/validator/v10#readme-baked-in-validations

Reply:
Thanks for pointing this out! I also ran into this document, but I could have very easily missed that